source/user-manual/capabilities/vulnerability-detection/offline-update.rst

```diff
@@ -341,7 +341,7 @@ Finally, you will have the feed divided into a succession of numbered files whos
 
     <provider name="nvd">
         <enabled>yes</enabled>
-        <path>/local_path/nvd-feed.*json$</path>
+        <path>/local_path/nvd-feed[[:digit:]]\{4\}\.json\.gz$</path>
```

Comment:
The documentation warns that the compressed files should be extracted, but then we use the compressed files in the regex. I validate that it was not necessary to extract them
![image](https://user-images.githubusercontent.com/13010397/173857203-2cd7ff55-9ca6-4634-a69e-ca5e61f06904.png)

Reply:
This seems to be the cause of the user's error: not decompressing the files when the regex expects a `.json`